Project/Modules/Blogs/Controllers/BlogsController.cs

```diff
@@ -0,0 +1,54 @@
+using blog_net_core.Project.Modules.Blogs.Model.Entities;
+using blog_net_core.Project.Modules.Blogs.Services;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace blog_net_core.Controllers
+{
+    [Route("blogs")]
+    [ApiController]
+    public class BlogsController : ControllerBase
+    {
+        private readonly IBlogService _blogService;
+
+        /// <summary>
+        /// The constructor of the BlogController.
+        /// </summary>
+        /// <param name="blogService"></param>
+        public BlogsController(IBlogService blogService)
+        {
+            _blogService = blogService;
+        }
+
+        /// <summary>
+        /// Get all Blogs.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
+        [Route("{id:int}")]
+        [HttpGet]
+        public async Task<Microsoft.AspNetCore.Mvc.ActionResult> Get(int id)
+        {
+            var blog = await _blogService.getBlogById(id); 
+            return Ok(blog);
```

Comment:
return the blog without the OK, you will need to change the return type of the fucntion.

Reply:
Resolved.